source/js/components/join/join.jsx

```diff
@@ -54,7 +54,6 @@ export default class JoinUs extends React.Component {
 
     let signupState = classNames({
       'row': true,
-      'py-3': true,
```

Comment:
I think this is what removes padding from the "Join Us" component on opportunity pages, which is a regression. Keep in mind that component has 3 different versions.